src/components/Settings/Password.js

```diff
@@ -1,23 +1,51 @@
 import React, { Component, Fragment } from "react";
 import { connect } from "react-redux";
-import { Input, Row, Col, Button } from 'antd';
-
+import { Input, Row, Col, Button, message } from 'antd';
+import {updatePassword} from '../../redux/actions/userActions';
 const InputGroup = Input.Group;
 
 class Password extends Component {
+  state ={
+    oldPassword: null,
+    newPassword: null,
+    confNewPassword: null
+  }
+  onChange = e=>{
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+  onClick = ()=>{
+    if(this.state.newPassword !== this.state.confNewPassword){
+      message.error("New password doesn't match");
```

Comment:
Change this to `Password does not match`

Reply:
Reset password is not working. Also no need to have an ongoing redux actions, set a loading state on the button as we have in login. Also use .then() to set the message to display, have a separate state for that.

Reply:
It is working fine for me , made the other change